src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -418,15 +448,7 @@ let main argv =
     let ignoreFiles = results.GetResult(<@ Ignore_Files @>, [])
     printInfo "Ignore Files: [%s]" (ignoreFiles |> String.concat ", ")
     let ignoreFiles = ignoreFiles |> List.map Glob
-    let configuration = results.TryGetResult <@ Configuration @>
-
-    let properties =
-        results.GetResults <@ Property @>
-        |> expandMultiProperties
-        |> fun props ->
-            configuration
-            |> Option.map (fun c -> List.append props [ "Configuration", c ])
-            |> Option.defaultValue props
+    let properties = getProperties results
```

Comment:
Can properties be combined with `--fsc-args`?

Reply:
I expect they can be combined but I'll test next week.

Reply:
My initial thinking is that they cannot. Properties would influence the project cracking I assume and we skip that entirely when using `--fsc-args`.

Reply:
As we discussed, while we could shoehorn some parts from the properties into the fsc-args, it would be against the idea of fsc-args, so let's disallow that.